app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -32,7 +33,7 @@
       %input.form-control{"type"         => "password",
                           "id"           => "textInput-markup",
                           "ng-required"  => "#{ng_reqd_password}",
-                          "placeholder"  => "#{placeholder_if_present("#{ng_model}.#{prefix}_password")}",
+                          "placeholder"  => "",
```

Comment:
@fryguy @martinpovolny she is trying to fix an issue that got introduced with angular conversion, it should not show dots in the password field when user does not have any password data saved yet.